doc/rho.1

```diff
@@ -0,0 +1,408 @@
+.TH rho 1 "August 10, 2017" "version 0.1" "rho User Commands"
+.SH NAME
+rho \- Easily discover and manage systems on your network.
+
+.SH SYNOPSIS
+rho
+.I command subcommand [options]
+
+.SH DESCRIPTION
+rho is a network discovery tool to identify the number of systems running on a network, their operating system, hardware, and release numbers for almost any Linux or Unix flavor. Being able to identify the systems running on the network is a vital component to managing licenses and renewals. Ultimately, discovery is part of the larger sysadmin task of managing inventories.
+.PP
+rho uses three configuration entries to manage the discovery process.
+.I Profiles
+define the network or subnet that is being monitored.
+.I Credentials
+contains the usernames, passwords or, alternatively, the SSH keys of the user as which discovery runs. There can be multiple network profiles and authentication credentials, used in any combination.
+.I Reports
+define the fields and output filename that will be run with a scan.
+.PP
+rho is an
+.I agentless
+discovery tool, so there is no need to install anything on multiple systems. OS discovery for the entire network is centralized to a single machine.
+.PP
+This man page covers the commands, subcommands and options for rho with basic usage information. For more detailed information and examples, including best practices, see the rho README.
+
+.SH USAGE
+rho performs four major tasks:
```

Comment:
Changed to three for now, but I think we'll need to do another update to man page to make sure we cover all commands. I'll open an issue for this.